locale/en/blog/vulnerability/openssl-and-zlib-vulnerability-assessment.md

```diff
@@ -0,0 +1,57 @@
+---
+date: 2022-10-24T17:00:15.000Z
```

Comment:
Note that the server runs on UTC -- at the time I'm writing this comment this time is ten minutes in the future 🙂.

Reply:
Increased 3 hours

Reply:
I was thinking about whether a date in the future would prevent the page showing up when the website is rebuilt, but maybe that only applies to the banner and the date for blog posts only affects the ordering in which they are listed?

Reply:
Does a PR gets merged with a date in the past? That's the usual workflow?

Reply:
Let's not worry about it for now -- we can always fix it up later if it's an issue.